test/e2e/dns_autoscaling.go

```diff
@@ -293,3 +342,34 @@ func waitForDNSConfigMapCreated(c clientset.Interface, timeout time.Duration) (c
 	}
 	return configMap, nil
 }
+
+func waitForDNSPodsRunningWithNum(c clientset.Interface, expNum int, timeout time.Duration) error {
```

Comment:
don't you want to wait for running + ready? is there a framework.util function you can leverage, this seems like a very common task (wait for pods with some label to enter running+ready in a given namespace). If there's no such function, can we make this a generic function and move it to util.go?